dashboard/app/main.go

```diff
@@ -592,6 +606,28 @@ func textFilename(tag string) string {
 	}
 }
 
+func fetchFixPendingBugs(c context.Context, ns, manager string) ([]*Bug, error) {
+	filter := func(query *db.Query) *db.Query {
+		query = query.Filter("Namespace=", ns).
+			Filter("Status<", BugStatusFixed)
```

Comment:
Does it work if you add `Filter("Commits>", "")`?
Dashboard pages become slower and slower over time.